cmd/bootstrap/cmd/finalize.go

```diff
@@ -170,10 +170,15 @@ func finalize(cmd *cobra.Command, args []string) {
 	constructRootResultAndSeal(flagRootCommit, block, stakingNodes, assignments, clusterQCs, dkgData)
 	log.Info().Msg("")
 
-	log.Info().Msg("copying internal private keys to output folder")
-	err := copyDir(flagInternalNodePrivInfoDir, filepath.Join(flagOutdir, model.DirPrivateRoot))
-	if err != nil {
-		log.Error().Err(err).Msg("could not copy private key files")
+	// copy files only if the directories differ
+	if flagInternalNodePrivInfoDir != flagOutdir {
+		log.Info().Msg("copying internal private keys to output folder")
```

Comment:
could we log the path of the `from` and `to` directory. We wanted to confirm the files have been copied over, but was looking at the `from` folder by mistake, and then thought the files were not copied.